python/doc/pyplots/P1LagrangeEvaluation.py

```diff
@@ -12,9 +12,8 @@
 cloud = ot.Cloud(mesh.getVertices(), values)
 cloud.setPointStyle("square")
 graph.add(cloud)
-graph.setColors(["blue", "red"])
 graph.setTitle("P1 Lagrange interpolation")
 
 fig = plt.figure(figsize=(10, 4))
 func_axis = fig.add_subplot(111)
-view = View(graph, figure=fig, axes=[func_axis], add_legend=False)
+View(graph, figure=fig, axes=[func_axis], add_legend=False)
```

Comment:
View(graph, figure=fig, axes=[func_axis], add_legend=False).show()

I think show() is necessary

Reply:
No it is not, this is designed to produce the plot on the API doc page. The reason it does not is that this page uses the "class" template instead of the "classWithPlot" template. @adutfoy to fix that please insert 
`:template: classWithPlot.rst_t`
above "P1LagrangeEvaluation" in `python/doc/user_manual/functions.rst` :

![image](https://github.com/user-attachments/assets/ec2f6898-0c21-47d8-85ab-148f1ad598fd)

Reply:
Maybe there are other classes that do not use "classWithPlot" in the library?